cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -102,6 +102,13 @@ spec:
                 mgr:
                   type: object
                   properties:
+                    count:
+                      type: integer
+                      minimum: 1
+                      maximum: 2
+                      default: 1
+                    allowMultiplePerNode:
```

Comment:
I don't know but I'd like to avoid exposing dev flags in the API. Perhaps, we could introduce something in the operator configmap? @BlaineEXE thoughts on this?

Reply:
I'm thinking the CR is actually an ok place for test settings. The CR is the way we allow customization of a cluster. If there are some settings that should only be used for testing, for example allowing a single node deployment, it seems to fit the pattern.

Reply:
It might fit the pattern but I don't think exposing dev flags on software intended for production is a good idea. But we have a precedent with the mons...

Reply:
Is a production use-case not the same as a production use-case for mons where there are multiple per node? Like having more redundancy in single-node environments?

Reply:
Yes it is the same as for mons, but the point is really that a single-node environment is not for production. Is there a better pattern that would make sense to support a test environment than adding test-only settings to the CR? I still vote for the CR settings.